adapters/InMobi/InMobiAdapter/InMobiMediatedNativeAppInstallAd.m

```diff
@@ -37,149 +37,157 @@ - (instancetype)initWithInMobiNativeAppInstallAd:(IMNative *)nativeAd
   }
   self = [super init];
   self.adapter = adapter;
-  // self.connector = connector;
   self.native = nativeAd;
-  NSData *data = [self.native.adContent dataUsingEncoding:NSUTF8StringEncoding];
+  NSData *data = [self.native.customAdContent dataUsingEncoding:NSUTF8StringEncoding];
   NSError *error = nil;
+  SEL inmobiMediatedNativeAppInstallAdSuccessful = @selector(inmobiMediatedNativeAppInstallAdSuccessful:);
 
   if (data) {
     self.nativeAdContentDictionary =
         [NSJSONSerialization JSONObjectWithData:data options:kNilOptions error:&error];
     if (![[self.nativeAdContentDictionary objectForKey:TITLE] length] ||
```

Comment:
Please update the below lines of code here. 
        (!self.native.adTitle) ||
        (!self.native.description) ||
        (!self.native.adCtaText) ||

Reply:
Please update all the  HeaderFiles with  @import GoogleMobileAds; For the reference please click on the below [link.](https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Facebook/FacebookAdapter/GADFBAdapterDelegate.m#L17)